lib/if2ip.c

```diff
@@ -149,9 +151,10 @@ if2ip_result_t Curl_if2ip(int af, unsigned int remote_scope,
 
                 continue;
               }
+
+              if (scopeid)
```

Comment:
No idea how that happaned. Probably my Visual Studio forgot that if goes without a space when moving the block.

Reply:
Fixed.